packages/cc/src/cc/ZWaveLRProtocolCC.ts

```diff
@@ -0,0 +1,238 @@
+import {
+	CommandClasses,
+	type DataRate,
+	type FLiRS,
+	type NodeInformationFrame,
+	type NodeType,
+	type ProtocolVersion,
+	ZWaveError,
+	ZWaveErrorCodes,
+	// parseBitMask,
+	encodeNodeInformationFrame,
+	parseNodeInformationFrame,
+	validatePayload,
+} from "@zwave-js/core";
+import type { ZWaveHost } from "@zwave-js/host";
+import {
+	type CCCommandOptions,
+	CommandClass,
+	type CommandClassDeserializationOptions,
+	gotDeserializationOptions,
+} from "../lib/CommandClass";
+import {
+	CCCommand,
+	commandClass,
+	expectedCCResponse,
+	implementedVersion,
+} from "../lib/CommandClassDecorators";
+import {
+	// type NetworkTransferStatus,
+	// WakeUpTime,
+	ZWaveLRProtocolCommand,
+	// parseWakeUpTime,
+} from "../lib/_Types";
+
+@commandClass(CommandClasses["Z-Wave Long Range Protocol"])
+@implementedVersion(1)
+export class ZWaveLRProtocolCC extends CommandClass {
+	declare ccCommand: ZWaveLRProtocolCommand;
+}
+
+@CCCommand(ZWaveLRProtocolCommand.NOP)
+export class ZWaveLRProtocolCCNOP extends ZWaveLRProtocolCC {}
+
+interface ZWaveLRProtocolCCNodeInformationFrameOptions
+	extends CCCommandOptions, NodeInformationFrame
+{}
+
+// BUGBUG: how much of this can we share with existing stuff? Can we use a ZWaveProtocolCCNodeInformationFrameOptions field to do the `isLongRange` stuff?
+// BUGBUG: how much can we share also with the Smart Start things below that are VERY close to this stuff?
```

Comment:
Same, can we leverage base classes and combine a lot of this with the existing `ZWaveProtocolCC`?